### PR TITLE
export energy metrics

### LINKED ISSUE #624: Export metrics according to available energy schedules
In order to calculate prices and CO2 emissions according to energy schedules, we need to record these metrics for each hour and weekday for further retrieval.

### PR DESCRIPTION
**Pull request by zubenkoivan**
closes #624

### CODE REVIEW

**Review by YevheniiSemendiak — APPROVED** [review]
lgtm!
one minor

File: test.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    if "n2" not in description_words:
+        continue
+    if not description_words.intersection(("core", "cpu", "vcpu")):
+        continue
+    if description_words.intersection(("sole", "tenancy", "custom")):
+        continue
+
+    print(description)
+    print(json.dumps(sku, indent=2))
+    print()
```
[the remark this answers is not in the document]
Comment (zubenkoivan) [reply test.py:60]: yes, sorry

File: Makefile [hunk 2]
```diff
@@ -24,13 +24,13 @@ test_unit:
...
 
 test_integration:
-	docker-compose -f tests/integration/docker/docker-compose.yaml pull -q
-	docker-compose -f tests/integration/docker/docker-compose.yaml up -d
+	docker compose -f tests/integration/docker/docker-compose.yaml pull -q
+	docker compose -f tests/integration/docker/docker-compose.yaml up -d
 	@$(WAIT_FOR_IT) 0.0.0.0:3000 -- echo "grafana is up"
 	@$(WAIT_FOR_IT) 0.0.0.0:8080 -- echo "platform-auth is up"
-	@pytest -vv --log-level=INFO tests/integration; \
+	@pytest -vv --log-level=INFO tests/integration -k test_api; \
```
Comment (dalazx) [inline Makefile:31]: is this intentional that we are skipping other tests?

**Review by dalazx — APPROVED** [review]

File: Makefile (hunk 2 above)
Comment (zubenkoivan) [reply Makefile:31]: No, sorry forgot to revert

**Pull request merged**